engine/userguide/storagedriver/aufs-driver.md

```diff
@@ -97,43 +97,58 @@ back to a "copy and unlink" strategy.
 
 ## Configure Docker with AUFS
 
+### Prerequisites
+
 You can only use the AUFS storage driver on Linux systems with AUFS installed.
 Use the following command to determine if your system supports AUFS.
 
-    $ grep aufs /proc/filesystems
+```bash
+$ grep aufs /proc/filesystems
+
+nodev   aufs
+```
 
-    nodev   aufs
+This output indicates the system supports AUFS. If you get no output, your system does
+not support AUFS. You can address this by upgrading your host system's kernel to 3.13
+or higher. It is recommended to intall the kernel headers when you upgrade.
 
-This output indicates the system supports AUFS. Once you've verified your
-system supports AUFS, you can must instruct the Docker daemon to use it. You do
-this from the command line with the `dockerd` command:
+### Configuration
 
-    $ sudo dockerd --storage-driver=aufs &
+When you have verified that you meet the prerequisites, instruct the Docker daemon to use
+AUFS by starting the Docker daemon with the flag `--storage-driver=aufs`:
 
+```bash
+$ sudo dockerd --storage-driver=aufs &
+```
 
-Alternatively, you can edit the Docker config file and add the
+To make the change permanent, you can edit the Docker configuration file and add the
 `--storage-driver=aufs` option to the `DOCKER_OPTS` line.
 
-    # Use DOCKER_OPTS to modify the daemon startup options.
-    DOCKER_OPTS="--storage-driver=aufs"
+```none
+# Use DOCKER_OPTS to modify the daemon startup options.
+DOCKER_OPTS="--storage-driver=aufs"
```

Comment:
not related, but looks like we need to update this as well, to use the daemon.json approach